src/backend/web/handlers/account.py

```diff
@@ -459,6 +461,75 @@ def mytba_event_post(event_key: EventKey) -> Response:
     )
 
 
+@blueprint.get("/mytba/eventteam/<int:team_number>")
+@require_login
+def mytba_eventteam_get(team_number: TeamNumber) -> str:
+    team = Team.get_by_id(f"frc{team_number}")
+    if not team:
+        abort(404)
+
+    user = none_throws(current_user())
+
+    event_teams = EventTeam.query(EventTeam.team == team.key).fetch(1000)
```

Comment:
This should use the cached query, probably - 

https://github.com/the-blue-alliance/the-blue-alliance/blob/34f00ad47800bc9695561deba46083dd3141e8d6/src/backend/common/queries/event_query.py#L99